tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
What's the difference between ES5 and ES2015?

Reply:
ES2015 is ES6, ES5 is ES2009.